src/plugin-manager.js

```diff
@@ -8,7 +8,7 @@ import {FrameworkConfiguration} from 'aurelia-framework';
  * @export
  * @interface PluginDefinition
  */
-export interface PluginDefinition {
+interface PluginDefinition {
```

Comment:
It was causing this repetition 
https://github.com/SpoonX/aurelia-config/blob/master/dist/aurelia-config.d.ts#L24

This should be the fix ? 
https://github.com/Microsoft/TypeScript/issues/3194#issuecomment-103080467